pytximport/core/_tximport.py

```diff
@@ -542,6 +542,29 @@ def tximport(
             uns=uns,
         )
 
+    if output_type == "summarizedexperiment":
+        from biocframe import BiocFrame
+        from summarizedexperiment import SummarizedExperiment
+
+        meta_obj = {
```

Comment:
The variable naming convention used throughout this project prefers full words and explicit names (even if slightly verbose) over abbreviations. In addition, "obj" does not convey information here. Something like "metadata" (short and simple), "run_metadata", "summarized_experiment_metadata", etc. may be preferable